tests/unit/lms/views/predicates/_lti_launch_test.py

```diff
@@ -16,55 +16,38 @@
 class TestDBConfigured:
     @pytest.mark.parametrize("value,expected", [(True, True), (False, False)])
     def test_when_theres_a_matching_assignment_config_in_the_db(
-        self, pyramid_request, value, expected, ModuleItemConfiguration
+        self, pyramid_request, value, expected
     ):
-        ModuleItemConfiguration.get_document_url.return_value = "test_document_url"
+        mic = factories.ModuleItemConfiguration()
         pyramid_request.params = {
-            "resource_link_id": "test_resource_link_id",
-            "tool_consumer_instance_guid": "test_tool_consumer_instance_guid",
+            "resource_link_id": mic.resource_link_id,
+            "tool_consumer_instance_guid": mic.tool_consumer_instance_guid,
         }
         predicate = DBConfigured(value, mock.sentinel.config)
 
         result = predicate(mock.sentinel.context, pyramid_request)
 
-        ModuleItemConfiguration.get_document_url.assert_called_once_with(
-            pyramid_request.db,
-            "test_tool_consumer_instance_guid",
-            "test_resource_link_id",
-        )
         assert result is expected
 
     @pytest.mark.parametrize("value,expected", [(True, False), (False, True)])
     def test_when_theres_no_matching_assignment_config_in_the_db(
-        self, pyramid_request, value, expected, ModuleItemConfiguration
+        self, pyramid_request, value, expected
     ):
-        ModuleItemConfiguration.get_document_url.return_value = None
         pyramid_request.params = {
             "resource_link_id": "test_resource_link_id",
             "tool_consumer_instance_guid": "test_tool_consumer_instance_guid",
         }
```

Comment:
These strings look like magic values, but I presume are actually irrelevant. Could we change them if so to say so. Something like:

```suggestion
        pyramid_request.params = {
            "resource_link_id": "missing_id",
            "tool_consumer_instance_guid": "missing_guid",
        }
```